modules/net-vpc/variables.tf

```diff
@@ -253,11 +253,12 @@ variable "shared_vpc_service_projects" {
 variable "subnets" {
   description = "Subnet configuration."
   type = list(object({
-    name                  = string
-    ip_cidr_range         = string
-    region                = string
-    description           = optional(string)
-    enable_private_access = optional(bool, true)
+    name                             = string
+    ip_cidr_range                    = string
+    region                           = string
+    description                      = optional(string)
+    enable_private_access            = optional(bool, true)
+    allow_subnet_cidr_routes_overlap = optional(bool, false)
```

Comment:
we typically use `null` as default, to avoid setting an attribute when it's not needed